docs/runbooks/OrphanedVirtualMachineInstances.md

```diff
@@ -1,77 +1,65 @@
 
 ## Impact
 
-When a node does not have a running `virt-handler` any VMI running on that node is considered orphaned. VMIs being orphaned means that they are no longer manageable.
+Orphaned VMIs cannot be managed.
 
 ## Diagnosis
 
-You can confirm the alert by finding which nodes your virt-handler pods are running on.
-
-```bash
-kubectl get pods --all-namespaces -o wide -l kubevirt.io=virt-handler
-```
-*Output:*
-```
-NAME                 READY   STATUS    RESTARTS   AGE  IP               NODE     NOMINATED NODE   READINESS GATES
-virt-handler-vhqsp   1/1     Running   0          4h   10.244.140.80    node02   <none>           <none>
-virt-handler-xd8jc   1/1     Running   0          4h   10.244.196.168   node01   <none>           <none>
-```
-
-Then check to see which nodes the VMIs are running on. If they are running on a node that a virt-handler pod does not exist on - those VMIs are orphaned.
-
-```bash
-kubectl get vmis --all-namespaces
-```
-
-*Output:*
-```
-NAMESPACE   NAME            AGE   PHASE       IP    NODENAME
-default     vmi-ephemeral   4s    Scheduled         node02
-```
+1. Check the status of the `virt-handler` pods to view the nodes on which they are running:
+  ```bash
+  $ kubectl get pods --all-namespaces -o wide -l kubevirt.io=virt-handler
+  ```
+2. Check the status of the VMIs to identify VMIs running on nodes that do not have a running `virt-handler` pod:
+  ```bash
+  $ kubectl get vmis --all-namespaces
+  ```
+3. Check the status of the `virt-handler` daemon:
+  ```bash
+  $ kubectl get daemonset virt-handler --all-namespaces
+  ```
+  Example output:
+  ```
+  NAME                  DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR            AGE
+  virt-handler          2         2         2       2            2           kubernetes.io/os=linux   4h
+  ```
+  The daemon set is considered healthy if the `Desired`, `Ready`, and `Available` columns contain the same value.
 
 ## Mitigation
 
-Check to see if the DaemonSet that controls the `virt-handler-*` pods is healthy.
-
-```bash
-kubectl get daemonset virt-handler --all-namespaces
-```
-
-*Output:*
-```
-NAME                    DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR            AGE
-virt-handler            2         2         2       2            2           kubernetes.io/os=linux   4h
-```
-
-The DaemonSet is considered healthy if Desired, Ready and Available are all the same number.
-
-
-### Unhealthy virt-handler DaemonSet
-Check the DaemonSets status to see what the issues are with deploying the pods.
-
-`kubectl describe daemonset virt-handler --all-namespaces`
-
-
-Another way to check the status is to get the object and read through its status object.
-
-`kubectl get daemonset virt-handler --all-namespaces -o yaml | jq .status`
+If the `virt-handler` daemon set is not healthy:
```

Comment:
@apinnick I think it should be DaemonSet

Reply:
also in the rest of the places in this runbooks

Reply:
No, it should be "daemon set".